AdminGUI/package.json

```diff
@@ -20,6 +20,7 @@
         "angular2-in-memory-web-api": "0.0.7",
         "basecode": "1.2.0",
         "es6-shim": "0.35.0",
+        "gulp-typescript": "^4.0.2",
```

Comment:
Should this be moved to `devDependencies` instead?

Reply:
Removed gulp-typescript because we won't use it anymore.